tsl/test/sql/continuous_aggs_drop_chunks.sql

```diff
@@ -0,0 +1,53 @@
+\c :TEST_DBNAME :ROLE_SUPERUSER
```

Comment:
Looks like you are missing a License header here

Reply:
Fixed.

Reply:
Added.